provisioning/Build.ps1

```diff
@@ -0,0 +1,87 @@
+<#
```

Comment:
I would say that it's heavily inspired by the windows scripts... The Provisioning.psm1 script is identical though. So you probably have a point